src/Task/Assets/loadTasks.php

```diff
@@ -1,41 +0,0 @@
-<?php
-namespace Robo\Task\Assets;
-
-trait loadTasks
-{
-    /**
-    * @param $input
-    * @return Minify
-    */
-    protected function taskMinify($input)
-    {
-        return new Minify($input);
```

Comment:
At the moment, I have these registered as 'taskMinify', and `task` adds the "task" part on the beginning (a sort of container namespacing). So, under this naming convention, this would be `return $this->task('Minify');` == `$container->get('taskMinify');`.  Could change this to `return $this->task('minify');` == $container->get('minify');` or whatever you think is best; no strong opinions here ATM.